### PR TITLE
un-relying on js getting automatically loaded

### PR DESCRIPTION
**Pull request by gluneau**
Renamed all js files that were named identically to their module names and explicitly called them in the getJSDependencies function.

Also fixed one pselect that was missing an array.

### CODE REVIEW

File: modules/acknowledgements/php/NDB_Menu_Filter_Form_acknowledgements.class.inc [hunk 1]
```diff
@@ -225,7 +225,7 @@ class NDB_Menu_Filter_Form_Acknowledgements extends NDB_Menu_Filter_Form
         return array_merge(
             $baseDeps,
             array(
-             $baseurl . '/acknowledgements/js/acknowledgements.js',
+             $baseurl . '/acknowledgements/js/acknowledgements_helper.js',
```
Comment (driusan) [inline modules/acknowledgements/php/NDB_Menu_Filter_Form_acknowledgements.class.inc]: Not that it makes much difference, but is there any reason for all the js file renames?

File: modules/reliability/php/NDB_Menu_Filter_reliability.class.inc [hunk 2]
```diff
@@ -310,7 +310,7 @@ class NDB_Menu_Filter_reliability extends NDB_Menu_Filter
     function _populate_reliability_table()
     { 
        $db =& Database::singleton();
-       $this->reliability_table= $db->pselect("SELECT CommentID, reliability_center_id FROM reliability");
+       $this->reliability_table= $db->pselect("SELECT CommentID, reliability_center_id FROM reliability", array());
```
Comment (driusan) [inline modules/reliability/php/NDB_Menu_Filter_reliability.class.inc]: These two changes seem unrelated to unrelying on automatic JS behaviour.

File: modules/acknowledgements/php/NDB_Menu_Filter_Form_acknowledgements.class.inc (hunk 1 above)
Comment (gluneau) [inline modules/acknowledgements/php/NDB_Menu_Filter_Form_acknowledgements.class.inc]: Yes, so there is no confusion with what gets automagically loaded.  Now that the files are not identical to the module name, they will not trigger the warning in the logs.

File: modules/reliability/php/NDB_Menu_Filter_reliability.class.inc (hunk 2 above)
Comment (gluneau) [inline modules/reliability/php/NDB_Menu_Filter_reliability.class.inc]: Yes, this change is not related, but I as watch the logs I fixed this missing array.  I will mention it in the pull request description.

File: modules/dashboard/php/NDB_Form_dashboard.class.inc [hunk 3]
```diff
@@ -398,6 +398,7 @@ class NDB_Form_Dashboard extends NDB_Form
             array(
              $baseurl . '/js/d3.min.js',
              $baseurl . '/js/c3.min.js',
+             $baseurl . '/dashboard/js/dashboard-helper.js',
```
Comment (johnsaigle) [inline modules/dashboard/php/NDB_Form_dashboard.class.inc]: Typo here.. I think it should be dashboard_helper.js!
